src/hub/controllers/OdometryController_Config.ts

```diff
@@ -23,7 +23,11 @@ const OdometryController_Config: SourceListConfig = {
         "Transform2d",
         "Transform3d",
         "Transform2d[]",
-        "Transform3d[]"
+        "Transform3d[]",
+        "DifferentialSample",
```

Comment:
I don't think it makes sense to support the non-array version of this as a trajectory type (this applies to the rest of the config too).